src/buildMathML.js

```diff
@@ -454,6 +454,11 @@ groupTypes.phantom = function(group, options, prev) {
     return new mathMLTree.MathNode("mphantom", inner);
 };
 
+groupTypes.not = function(group, options) {
+    // TODO: it's an operator first and find the correct "not" version of that operator
+    return buildGroup(group.value.body, options);
```

Comment:
In the case where there isn't a "not" version of whatever's in the body, I'll add the solidus in manually.  This is what MathJax does:
```
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mrow class="MJX-TeXAtom-REL">
    <mpadded width="0">
      <mtext>&#x29F8;</mtext>
    </mpadded>
  </mrow>
  <mrow class="MJX-TeXAtom-ORD">
    <mi>a</mi>
    <mi>b</mi>
    <mi>c</mi>
  </mrow>
</math>
```